pkg/storage/unified/apistore/prepare.go

```diff
@@ -56,6 +56,15 @@ func (s *Storage) prepareObjectForStorage(ctx context.Context, newObject runtime
 		obj.SetUID(types.UID(uuid.NewString()))
 	}
 
+	if s.opts.RequireDeprecatedInternalID {
+		// nolint:staticcheck
```

Comment:
nit, but as someone reading the code i find it helpful if these have comments with them explaining why they are needed. that way they aren't preserved if things change that don't require the skip, or if someone copies the code without knowing the context of why the skip was needed in the first place